### PR TITLE
Keep PostRecommendation from Showing Posts from Negative Rep Authors

### LINKED ISSUE #1539: recommended posts include posts from low reputation users
#### Expected behavior

The recommended posts should not show posts from users with negative rep

#### Actual behavior

Posts are recommended from authors with negative rep

#### How to reproduce

Visit a post from a negative rep author who has multiple posts

### PR DESCRIPTION
**Pull request by caleblogan**
Fixes #1539 

Changes:
- Add filter to PostRecommendation for negative rep authors

### CODE REVIEW

File: src/client/components/Sidebar/PostRecommendation.js [hunk 1]
```diff
@@ -73,6 +74,7 @@ class PostRecommendation extends Component {
   getFilteredPosts = () => {
     const { match } = this.props;
     return this.state.recommendedPosts
+      .filter(post => formatter.reputation(post.author_reputation) > -1)
       .filter(post => post.permlink !== match.params.permlink)
```
Comment (jm90m) [inline src/client/components/Sidebar/PostRecommendation.js]: you don't need to iterate through the list again, you can create 1 filter function above that handles both filtering cases here

**Comment by jm90m** [comment]
I am not too sure if #1539, is what we want, @bonustrack has final say on these types of things.

**Comment by caleblogan** [comment]
Do you mean not adding this feature? Or something more like if the author has negative rep, suggest posts from one of the interesting authors?

**Comment by jm90m** [comment]
@caleblogan yeah not adding this feature, when people create issues for features like this, @bonustrack has to approve of it

**Comment by caleblogan** [comment]
Yeah that makes sense.

**Comment by jm90m** [comment]
Thanks for the contribution, me and @bonustrack talked about it, and we have approved this.

**Pull request merged**